core/templates/pages/exploration-player-page/modals/flag-exploration-modal.component.html

```diff
@@ -38,7 +38,8 @@ <h4>
         {{ 'I18N_PLAYER_REPORT_MODAL_BODY_ADDITIONAL_DETAILS' | translate }}
         <textarea rows="3" cols="50"
                   [(ngModel)]="flagMessage" [ngModelOptions]="{standalone: true}"
-                  class="form-control e2e-test-report-exploration-text-area" oppiaFocusOn="flagMessageTextarea">
+                  class="form-control e2e-test-report-exploration-text-area" oppiaFocusOn="flagMessageTextarea"
+                  style="height: 210px; resize: none;" >
```

Comment:
Could you please apply these styles by specifying these two attributes in a separate rule instead (you can create a new class for this)? We avoid inline styling.